src/CacheItemInterface.php

```diff
@@ -72,12 +72,12 @@ public function isHit();
      * @return static
      *   The invoked object.
      */
-    public function set($value);
+    public function set(mixed $value): static;
 
     /**
      * Sets the expiration time for this cache item.
      *
-     * @param \DateTimeInterface|null $expiration
+     * @param ?\DateTimeInterface $expiration
```

Comment:
I think it should stay as is: https://docs.phpdoc.org/3.0/guide/references/phpdoc/types.html

Reply:
Doesn't that make it inconsistent with the native types, then?

(PHPDoc's own spec is well over a decade old at this point, no?  That's the reason PSR-5 and friends, er, tried to exist...)

Reply:
No. It's not inconsistent with PHP 8 union types and it's current PHPDocumentor 3 syntax that has wide support among IDEs and static code analysis tools.